src/main/java/com/evuv/builders/ExpressionBuilder.java

```diff
@@ -68,4 +59,13 @@
 		ev.setPropertyName(name);
 		return new SimplePropertyExpression<T>(ev);
 	}
+
+	public static <T> InExpression<T> in(Expression<T> left, Expression<Collection<T>> right) {
```

Comment:
Its better if the builder will return Expression<T>  interface if possible instead of InExpression class